two_factor/gateways/twilio/views.py

```diff
@@ -23,13 +24,21 @@ class TwilioCallApp(View):
         'press_a_key': '<?xml version="1.0" encoding="UTF-8" ?>'
                        '<Response>'
                        '  <Gather timeout="15" numDigits="1" finishOnKey="">'
-                       '    <Say language="%(locale)s">%(press_a_key)s</Say>'
+                       '    <Say language="{{ locale }}">{{ press_a_key }}</Say>'
                        '  </Gather>'
-                       '  <Say language="%(locale)s">%(no_input)s</Say>'
+                       '  <Say language="{{ locale }}">{{ no_input }}</Say>'
                        '</Response>',
         'token': '<?xml version="1.0" encoding="UTF-8" ?>'
                  '<Response>'
-                 '  <Say language="%(locale)s">%(token)s</Say>'
```

Comment:
as we switch to Django templates anyway, I'd say moving it to file based templates might be more consequent. but besides that, I had reverted the same approach with Django templates :)